src/dvclive/live.py

```diff
@@ -247,6 +248,12 @@ def end(self):
                 logger.warning("`post_to_studio` `done` event failed.")
         else:
             self.make_report()
+        if dvc_installed():
+            from dvc.repo import Repo
+
+            from dvclive.utils import random_name
+
+            Repo().experiments.save(name=random_name())
```

Comment:
When running this for the first time, either DVCLive or DVC needs to add the DVCLive outputs to Git, right?

Reply:
Yes. I am giving it a try on DVC side first